core/libs/editor/src/lib/editor-toolbar/components/help/document.component.html

```diff
@@ -29,7 +29,7 @@ <h2 mat-dialog-title>Start modeling</h2>
     <p>
       Please refer to the
       <a href="{{ SAMMDocumentationLink }}" target="_blank" rel="noopener">Specification</a>
-      for general help about the Digital Twin System and the Aspect Meta Model.
+      for general help about the Digital Twin System and the Aspect Meta Model as part of the Nexeed Digital Twin System.
```

Comment:
I think this change should be reverted.
Also, replace "Aspect Meta Model" with "Semantic Aspect Meta Model".